File: swagger/models/substitute_class_teacher_response.py
from .base_model import BaseModel

from .substitute_teacher_class import SubstituteTeacherClass

class SubstituteClassTeacherResponse(BaseModel):
	"""
	no description available

	Attributes:

		class
		Contains information about the class that is being assigned a
		substitute teacher.
	"""

	input_map = {
		'Class': ('class', Class),
		}

	output_map = {
		'class': 'Class',
		}

	class = None

File: swagger/Models/SubstituteClassTeacherResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class SubstituteClassTeacherResponse extends BaseModel
{
	/**
	 * Contains information about the class that is being assigned a
	 * substitute teacher.
	 * 
	 * @var SubstituteTeacherClass 
	 */
	public $class;

	protected function getInputMap()
	{
		return [
			'Class' => ['class', SubstituteTeacherClass::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'class' => ['Class', SubstituteTeacherClass::class, null],
			];
	}
}
